lib/api/webSockets.js

```diff
@@ -58,12 +58,19 @@ function connect(connectionOps = {}) {
 		}
 
 		if (content.type === 'console' && content.data) {
-			const method = {
-				info: 'info',
-				log: 'log',
-			}[content.subtype] || 'debug';
-
-			return logger[method](`|L| ${method}:`, content.data.join('\n'));
+			const logType = {
+				info: '|AI|',
```

Comment:
these should have been constants (probably in opType.js) along with other used markup.